datafusion/core/tests/sqllogictests/postgres/test_files/simple_except.slt

```diff
@@ -0,0 +1,27 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+query I
+SELECT * FROM (
+    SELECT c2
+    FROM aggregate_test_100_by_sql t1
+    EXCEPT
+    SELECT c2
+    FROM aggregate_test_100_by_sql t2
+    WHERE c2 IN (3, 4)
+) s
+ORDER BY c2
```

Comment:
I think you have to include expected output here to get sqllogictest to verify the output

```suggestion
query I
SELECT * FROM (
    SELECT c2
    FROM aggregate_test_100_by_sql t1
    EXCEPT
    SELECT c2
    FROM aggregate_test_100_by_sql t2
    WHERE c2 IN (3, 4)
) s
ORDER BY c2
---
foo bar
```

Reply:
Well, I tried to follow the approach of the current `integration-tests` implementation. 

The expected output is generated based on the Postgres execution here https://github.com/apache/arrow-datafusion/blob/d5369d0b05ce3b00c5a72c90eac567d7494fe20b/datafusion/core/tests/sqllogictests/src/main.rs#L97-L104. 

And then Datafusion does comparison to that output https://github.com/apache/arrow-datafusion/blob/d5369d0b05ce3b00c5a72c90eac567d7494fe20b/datafusion/core/tests/sqllogictests/src/main.rs#L106-L108

Reply:
Got it -- sorry I was slightly confused when I first looked at this PR

Reply:
The results are now explicit and Postgres runs only when an env var is set.